fabric-networking-v0/src/main/java/net/fabricmc/fabric/impl/networking/PacketDebugOptions.java

```diff
@@ -18,6 +18,7 @@
 
 public final class PacketDebugOptions {
 	public static final boolean DISABLE_BUFFER_RELEASES = System.getProperty("fabric.networking.broken.disableBufferReleases", "false").equalsIgnoreCase("true");
+	public static final boolean WARN_UNREGISTERED_PACKETS = System.getProperty("fabric.networking.warnUnregisteredPackets", Boolean.toString(PacketRegistryImpl.LOGGER.isDebugEnabled())).equalsIgnoreCase("true");
```

Comment:
Could [`Boolean.getBoolean()`](https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html#getBoolean-java.lang.String-) be used here?

Reply:
I did not use it because it does not support a default value (which, in my case, is whether the logger's debug is enabled)

Hope the `isDebugEnabled` call isn't too expensive so that I don't need to wrap it into a function

Reply:
Ah yeah, there's no default value support. It could have been used for `DISABLE_BUFFER_RELEASES`, I guess.

Reply:
yep, but i don't think i will touch this in this pr